### ISSUE TITLE
[Vite] Pre-transform error: Failed to load url /popup.js

### ISSUE THREAD

**Issue by aklinker1** [wxt-dev/wxt#421]
### Describe the bug

This error up when running `pnpm dev` with the vanilla template.

```
WXT 0.16.3                                                                                                     2:04:15 PM
✔ Started dev server @ http://localhost:3000                                                                  2:04:15 PM
ℹ Pre-rendering chrome-mv3 for development with Vite 5.0.12                                                   2:04:15 PM
⠸ [2/2] popup2:04:15 PM [vite] Pre-transform error: Failed to load url /popup.js (resolved id: /popup.js). Does the file exist?
✔ Built extension in 430 ms                                                                                   2:04:15 PM
  ├─ .output/chrome-mv3/manifest.json  849 B    
  ├─ .output/chrome-mv3/popup.html     532 B    
  ├─ .output/chrome-mv3/background.js  125.66 kB
  ├─ .output/chrome-mv3/popup.js       7.43 kB  
  ├─ .output/chrome-mv3/icon/128.png   3.07 kB  
  ├─ .output/chrome-mv3/icon/16.png    559 B    
  ├─ .output/chrome-mv3/icon/32.png    916 B    
  ├─ .output/chrome-mv3/icon/48.png    1.33 kB  
  ├─ .output/chrome-mv3/icon/96.png    2.37 kB  
  └─ .output/chrome-mv3/wxt.svg        1.07 kB  
Σ Total size: 143.79 kB                                 
✔ Opened browser in 680 ms
```

It doesn't seem to break anything... Likely related to the the breaking change in `0.16.0`, where we moved the entrypoints around? Still need to do a bisect to be sure.

> "In https://github.com/wxt-dev/wxt/pull/398, HTML pages' JS entrypoints in the output directory have been moved."
>
> &ndash; https://github.com/wxt-dev/wxt/releases/tag/v0.16.0

### To Reproduce

```sh
pnpx wxt@latest init repro --pm pnpm --template vanilla
cd repro
pnpm i
pnpm dev
```

### Expected behavior

This error should not be logged.

### Screenshots

<img width="665" alt="Screenshot 2024-02-05 at 2 08 40 PM" src="https://github.com/wxt-dev/wxt/assets/10101283/12eab732-2e6f-4d87-8cbc-7b5c78661d64">

### Environment

```
  System:
    OS: macOS 14.2.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 3.69 GB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.16.1 - ~/.asdf/installs/nodejs/18.16.1/bin/node
    Yarn: 1.22.19 - ~/.asdf/installs/nodejs/18.16.1/bin/yarn
    npm: 9.5.1 - ~/.asdf/plugins/nodejs/shims/npm
    pnpm: 8.9.0 - ~/.asdf/installs/nodejs/18.16.1/bin/pnpm
  Browsers:
    Chrome: 121.0.6167.139
    Safari: 17.2.1
  npmPackages:
    wxt: ^0.16.0 => 0.16.3
```

**Comment by qiweiii** [comment]
I also encountered this error, the hmr stoped working

```
[2/5] content11:04:04 PM [vite] Pre-transform error: Failed to load url /options.js (resolved id: /options.js). Does the file exist?
```

does this mean it happens when build content script?

**Comment by aklinker1** [comment]
Released fix in [`v0.16.11`](https://github.com/wxt-dev/wxt/releases/tag/v0.16.11)
